lib/broadway.ex

```diff
@@ -498,18 +498,20 @@ defmodule Broadway do
   `test/test_helper.exs`:
 
       defmodule BroadwayEctoSandbox do
+        alias MyApp.Repo
+
         def attach(Repo) do
```

Comment:
I think what we want to do instead is this change?

```suggestion
        def attach(repo) do
```